src/math-plugin.ts

```diff
@@ -33,8 +33,8 @@ function createMathView(displayMode:boolean){
 
 		// set up NodeView
 		let nodeView = new MathView(
-			node, view, getPos as (() => number), 
-			{ katexOptions : { displayMode, macros: pluginState.macros } },
+			node, view, getPos as (() => number),
+			{ tagName: displayMode ? 'math-display' : 'math-inline', katexOptions : { displayMode, macros: pluginState.macros } },
```

Comment:
I keyed this off of the display mode, which makes it more explicit and self contained?